frontend/src/components/worksheets/items/TableItem/BundleRow.js

```diff
@@ -165,8 +165,13 @@ class BundleRow extends Component {
             }
             if (Array.isArray(rowContent) && rowContent.length === 3) {
                 // Cell is a bundle genpath triple -- see is_bundle_genpath_triple() in backend.
-                // This means that the cell is only briefly loaded.
-                rowContent = <span style={{ color: 'grey' }}>Loading...</span>;
+                if (rowContent[0] === 'Forbidden') {
```

Comment:
Rather than returning `["Forbidden", "Forbidden", "Forbidden"]`, would it make more sense to return a single `["Forbidden"]` and then add a `if (rowContent.length === 1)` condition for this?

Reply:
updated.